lib/Test/ConchTmpDB.pm

```diff
@@ -40,8 +40,8 @@ sub mk_tmp_db {
 		$dbh->do('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";') or die;
 		$dbh->do('CREATE EXTENSION IF NOT EXISTS "pgcrypto";') or die;
 
-		for my $file ( io->dir("sql/migrations")->sort->glob("*.sql") ) {
-			$dbh->do( $file->all ) or die;
+		for my $file (sort glob('sql/migrations/*.sql')) {
```

Comment:
would this maybe be easier as `$dbh->do($_->slurp_utf8) or BAIL_OUT('Test SQL load failed') for sort path('sql/migrations)->children('*.sql')`?

Reply:
I would meh, but I do usually like removing unnecessary additional scopes.